lib/datasource/remote/api/rates_repository.dart

```diff
@@ -16,15 +16,44 @@ class RatesRepository extends NetworkRepository {
         .catchError((error) => mapHttpError(error));
   }
 
-  Future<Result<RateModel>> getUSDRate() async {
+  Future<Result<RateModel>> getSeedsRate() async {
     print('[http] get seeds rate USD');
 
     final request = '{"json":true,"code":"tlosto.seeds","scope":"tlosto.seeds","table":"price"}';
 
     return http
         .post(Uri.parse('$baseURL/v1/chain/get_table_rows'), headers: headers, body: request)
         .then((http.Response response) => mapHttpResponse<RateModel>(response, (dynamic body) {
-              return RateModel.fromJson(body);
+              return RateModel.fromSeedsJson(body);
+            }))
+        .catchError((error) => mapHttpError(error));
+  }
+
+  Future<Result<RateModel>> getTelosRate() async {
+    print('[http] get telos rate USD');
+
+    final params = '''
```

Comment:
please use `createRequest` method.

Reply:
Once again Gery saves the day... fixed this and found I was actually parsing the rate wrong

Thanks 🙏